processors/osf/processor.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+import logging
+import requests
+import datetime
+from .. import base
+logger = logging.getLogger(__name__)
+
+
+# Module API
+
+def process(conf, conn):
+
+    # Log started
+    logger.info('Started trials export')
+
+    # Export trials
+    # http://jamdb.readthedocs.io/en/latest/index.html
+    count = 0
+    limit = 100
+    token_ttl_seconds = 30*60  # api has limit 60*60 seconds
+    token_issued_time = datetime.datetime.now()
+    session = requests.Session()
+    for trials in _read_trial_groups(conn, limit=limit):
+
+        # Ensure authentificated
```

Comment:
There're a few others `authentificate`, so it'll be easier to do a search/replace